app/src/main/java/org/intelehealth/app/utilities/NotificationSchedulerUtils.kt

```diff
@@ -61,28 +61,31 @@ class NotificationSchedulerUtils {
 
         fun parseDateTimeToTimestamp(input: String): Long {
             return try {
-                val formatter = SimpleDateFormat("yyyy-M-d, 'Time':HH:mm", Locale.getDefault())
-                val date = formatter.parse(input)
```

Comment:
Use DateTimeUtils or create date format related methods inside the DateTimeUtils class